workbench-core/accounts/src/services/projectService.test.ts

```diff
@@ -103,6 +105,44 @@ describe('ProjectService', () => {
       expect(actualResponse).toEqual({ data: [] });
     });
 
+    test('list projects as IT Admin when no projects exist', async () => {
+      // BUILD
+      const queryItemResponse: QueryCommandOutput = {
+        $metadata: {}
+      };
+      const user: AuthenticatedUser = {
+        id: 'user-123',
+        roles: ['ITAdmin']
+      };
+
+      // mock getUserGroups--TODO update after dynamic AuthZ intergration
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      jest.spyOn(ProjectService.prototype as any, '_mockGetUserGroups').mockImplementation(() => ['ITAdmin']);
```

Comment:
possible to use `as unknown`?

Reply:
This will be taken out after dynamic authZ so I will leave it in for now